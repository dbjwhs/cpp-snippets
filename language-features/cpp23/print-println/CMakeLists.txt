# MIT License
# Copyright (c) 2025 dbjwhs

cmake_minimum_required(VERSION 3.31)
project(cpp23_print_println)

set(CMAKE_CXX_STANDARD 23)

add_executable(cpp23_print_println cmake-build-debug/print_println.cpp)
