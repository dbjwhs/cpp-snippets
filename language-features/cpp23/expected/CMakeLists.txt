# MIT License
# Copyright (c) 2025 dbjwhs

cmake_minimum_required(VERSION 3.31)
project(cpp23_expected)

set(CMAKE_CXX_STANDARD 23)

add_executable(cpp23_expected src/expected.cpp)
